test/functional/test_runner.py

```diff
@@ -110,6 +110,7 @@
     'rpcnamedargs.py',
     'listsinceblock.py',
     'p2p-leaktests.py',
+    'server.py',
```

Comment:
I took as an example the `net.py` test file that referred to tests in `rpc/net.cpp`. Since the uptime command is in the `server.cpp` - although there are no tests for this file - I chose to create a `server.py` test file. Do you still think it's best to use `uptime.py`?